### PR TITLE
Intrepid overhaul and additions

### PR DESCRIPTION
**Pull request by kyres1**
This adds a new hull and redoes basically the entire Intrepid from the ground up. The aim with this PR is to replicate its current capabilities/features at a minimum, and more if possible. More equipment, more space, better room layout. To top it off, brand spanking new sprites for the exterior.

Also notably, the pieces in this (and the Canary's PR) were cut up and there are more generic wall pieces now. This means non-spriter devs can remap it without it looking ridiculous ingame.
image

intrepid hangar umu

(ignore goofy resized oven)
image

This layout also pushes the size of shuttles to its workable limit, probably, but every landing site I've tested so far (exoplanets and open space) seems to work just fine.

New features include,
- Crew quarters with four bunk beds (since it's an expedition shuttle and soon we won't be required to return in 2 hours every shift)
- More general and emergency supplies (for less prep time and more fun time)
- A spare static air tank and more inbuilt pumps/valves to facilitate better gas usage (since it's an expedition shuttle)
- A more spacious cargo hold.
- A basic sit-on-your-butt-and-relax area with a compact oven for heating up food. Basic rations and stuff included.
- A washroom. See above. 
- No more welding tank, goodbye firefights that vent the entire first deck.
- External and internal refuel/air resupply connector ports.
- Not to mention the remaining spritework, hull and everything else.

Downsides include,
- Two less softsuits because I could not fit the other two for the fucking life of me.
- Arguably not a downside, the west airlock is gone for sake of having more space internally and to make the ship less dangerous overall. Less points of entry the better.
- This shit.
image

The lighting issue is something I have zero capability of fixing, but hopefully might be tackled later down the line. I am moreso hoping it's a simple fix.

### CODE REVIEW

**Comment by kyres1** [comment]
!review

**Comment by Greenjoe12345** [comment]
ooh, this is pretty, one concern however is the fact it has less seating, which is problematic when doing stuff like visiting the mall station with a bunch of crew

**Comment by kyres1** [comment]
> ooh, this is pretty, one concern however is the fact it has less seating, which is problematic when doing stuff like visiting the mall station with a bunch of crew

It has more seats in total and it shouldn't really be used for bulk transport of a horde of people without some minor renovations honestly. Handrails are also peppered everywhere too

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by kyres1** [comment]
> Overall I think it's an improvement but I agree with greenjoe about the seating. The intrepid is mostly for science to be doing away missions or to transport a lot of crew - not to live on comfortably or be a hangout spot for people. I think the mini kitchen and dorm are just wasted space.
> 
> The holopad could be moved to the general seating area if you care at all about fixing the seating issue, since most rounds where it's relevant, you want most the people onboard to be able hear what's going on.
> 
> My last concern is that it takes the ship 500 years to cycle the airlock, and having the smaller side one was handy when there was a hurry to get on or off - that said it looks like you added an extra vent so it might not matter.

Like I said above, seating hasn't changed. 

To fix the airlock issues, I've already added manual airlock controllers to the interior and exterior of the airlock. That way it can be forced if it's not cycling properly, and if it isn't, players can at least tell something is wrong instead of waiting twenty minutes to find out via the pressure sensor. There's blast doors on the exterior as well in case the ship still needs to be locked up.

As for the rest, the intrepid is not really designated to be a people-mover by nature. In the PR I'm cooking with the Spark overhaul similar to this one, I'll be tackling that separately by giving the crew more ways to get to/from the Intrepid or Spark's landing zones. Until then, this PR aims to ensure the Intrepid is a solid place for a home-away-from-home type of gameplay. Like a space RV!

**Comment by Llywelwyn** [comment]
Not sure if it's feasible or too much of a pain in the ass, but one thing that's really nice about the current layout is that whoever's piloting can talk with all the crew who're buckled in, but with this layout it's a bit more isolated. 

Maybe it would be cool if there were some railings in the area directly outside the helm, just so people could strap in there and still be able to interact with the pilots/holocall during takeoff/landing?

image

File: code/modules/cooking/machinery/cooking_machines/oven.dm [hunk 1]
```diff
@@ -122,6 +122,24 @@
 		return
 	..()
 
+/obj/machinery/appliance/cooker/oven/small
+	name = "compact oven"
+	desc = "A lightweight, small oven. Doesn't hold much, but it cooks just fine."
+	density = 0
+	anchored = 0
```
Comment (Greenjoe12345) [inline code/modules/cooking/machinery/cooking_machines/oven.dm:129]: it's designed to be unanchored?

Comment (kyres1) [reply code/modules/cooking/machinery/cooking_machines/oven.dm:129]: It's basically the size of a microwave so yeah

**Comment by kyres1** [comment]
> Not sure if it's feasible or too much of a pain in the ass, but one thing that's really nice about the current layout is that whoever's piloting can talk with all the crew who're buckled in, but with this layout it's a bit more isolated.
> 
> Maybe it would be cool if there were some railings in the area directly outside the helm, just so people could strap in there and still be able to interact with the pilots/holocall during takeoff/landing? I don't know much about mapping but might be an easy solution that doesn't require actually moving stuff.
> 
> image

(you can quote me on this) I plan to make an Intrepid or expedition-specific channel for radio comms in my next PR (the one that'll touch the Spark), but to help facilitate this feedback I added more intercomms just now to this one. Every room and seat should have access to the cockpit's intercomm through, well, general intercomms.

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**